lib/pushmi_pullyu/cli.rb

```diff
@@ -38,7 +38,7 @@ def run
       end
     # rubocop:disable Lint/RescueException
     rescue Exception => e
-      Rollbar.error(e)
+      Rollbar.error(e, use_exception_level_filters: true)
```

Comment:
We can probably make this global? Not sure how many `Rollbar.error` calls we doing in pushmi_pullyu (might just be this one, but if someone ever decides to come in and add another one, they don't have to forgot adding this `use_exception_level_filters: true` call).

Can add it right to the config:

```
Rollbar.configure do |config|
  config.use_exception_level_filters = true
end
```

(oddly this is not in the documentation, but heres a PR for the commit: https://github.com/rollbar/rollbar-gem/pull/588)

Reply:
Ah the PR example is wrong?

Correct usage is `use_exception_level_filters_default = true`

https://github.com/rollbar/rollbar-gem/blob/master/docs/configuration.md#use_exception_level_filters_default